### DISCUSSION TITLE
which database [Oct 19]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4183] [Discord Threads]
> 💁 **o1lab**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Derren**  
#### MySQL 5.7.38 (AWS RDS)   
####    
#### Don't think it took half an hour but that is when I happened to notice it was correct after doing other work.   
#### It doesn't take a couple of minutes, but there lots of "many-to-many" tables (not my design unfortunately).  I now know I need to wait, then it is okay.  
#  
> 💁 **o1lab**  
#### its should not be the case.    
####    
#### We'd issue with old Microsoft SQL server dbs there was time it took to show tables. Not anytime on MySQL  
#  
> 💁 **o1lab**  
#### so to clarify  
#  
> 💁 **o1lab**  
#### enabling m2m you are able to see all things fine ?  
#  
> 🙋‍♂️ **Derren**  
#### The process (repeated this morning)   
#### 1.  I created a new project to an external MySQL database, clicked Submit.   
#### 2.  Then after a (guess) couple of minutes, there was a message about an error.  The create project screen was still there.  I cancelled that, and my new project was listed.   
#### 3.  Click on the new project and 541 tables are listed.  (expect to see 184 tables).   
#### 4.  Exit the project and go into project after 5 minutes.  210 tables.   
#### 5.  Exit the project and go into the project after another 5 minutes.  184 tables.   
#### Now if I enable the m2m, I see 546 tables.   
#### Disabling m2m because would normally not want to see those tables.  
#  
> 💁 **o1lab**  
#### and are you on latest version ?  
#  
> 💁 **o1lab**  
#### 0.97.0  
#  
> 🙋‍♂️ **Derren**  
#### Yes, this is the result of "Copy Project Info"   
#### Node: **v12.22.12**   
#### Arch: **x64**   
#### Platform: **linux**   
#### Docker: **false**   
#### Database: **mysql2**   
#### ProjectOnRootDB: **false**   
#### RootDB: **mysql2**   
#### PackageVersion: **0.97.0**  
#  
> 💁 **o1lab**  
#### cool  
#  
> 💁 **o1lab**  
#### Will DM you  
#  
> 🙋‍♂️ **wkw**  
#### <@861694170401996902> can you double confirm the number of the expected items in table list (including tables + 4 views)  
#  
> 🙋‍♂️ **wkw**  
#### i can see 360 of them are mm (those started with `_`)  
#  
> 🙋‍♂️ **wkw**  
#### I got 186 in the table list tho  
#  
> 🙋‍♂️ **Derren**  
#### <@495466997304262658> should be 186  
#  
> 🙋‍♂️ **wkw**  
#### ok  
#  
> 🙋‍♂️ **wkw**  
#### in step 2,    
####    
#### > 2.  Then after a (guess) couple of minutes, there was a message about an error.    
####    
#### can you share the error message or error logs  
#  
> 🙋‍♂️ **wkw**  
#### i followed your steps and didn't see any errors. it takes around a min or two to navigate to the project page with 186 items in table list  
#  
> 🙋‍♂️ **wkw**  
#### exit the project then go back after some time, it s still 186  
#  
> 🙋‍♂️ **Derren**  
#### It was a flash of an error on the front end, I can't remember the wording.  There is nothing in the log file.  
#  
> 🙋‍♂️ **wkw**  
#### any useful info can be seen in cloudwatch?  
#  
> 🙋‍♂️ **wkw**  
#### or can u delete the project and retry it again  
#  
> 🙋‍♂️ **Derren**  
#### okay, it is fast on your side then it seems.   
#### MySQL which version?  
#  
> 🙋‍♂️ **Derren**  
#### I'll delete one of the projects and do it again.   
#### I'll check CloudWatch soon, just in a meeting now.  
#  
> 🙋‍♂️ **Derren**  
#### there wasn't really anything useful though.  
#  
> 🙋‍♂️ **wkw**  
#### im using 8.0.28 on my local machine tho  
#  
> 🙋‍♂️ **Derren**  
#### okay, we're using 5.7  
#  
> 🙋‍♂️ **Derren**  
#### There are no errors/warnings or anything suspect in the CloudWatch log for this morning/last night.   
####    
#### This time round when I created the project (now), I got a popup error (and it appears in the CloudWatch log) with "Knex: Timeout acquiring a connection. The pool is probably full. Are you missing a .transacting(trx) call?"   
####    
#### That is definitely not the error from before though.  Pretty sure that was a timeout error in the browser.   
####    
#### I can try the project create again later.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1029683977969217546)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
